lib/testLauncher/SuitestLauncher.js

```diff
@@ -180,6 +181,15 @@ class SuitestLauncher {
 							execPath: normalize(process.execPath),
 						}).on('close', resolve);
 
+						registerProcess(child);
+					} else if (repl) {
```

Comment:
@givehug so we can be in debug or in repl is there a validation for that. It may be confusing that presents of `inspect` in suitest args prevent repl mode

Reply:
There is no validation now. Yes it can be one of two only. At least in VS code repl can't be opened in debug console.

Reply:
@givehug do you think it can be useful to have debug and repl mode at same test run. The is also a chrome devtools which should not block repl

Reply:
It could be useful. We should test it with Chrome. If it's not possible to combine debug and repo modes there as well - we should warn user about it and close the process.

Reply:
yes it is possible